frontend/mr-admin-flate/src/components/tabell/AvtaleTabell.tsx

```diff
@@ -126,24 +126,33 @@ export const AvtaleTabell = () => {
     return <Laster size="xlarge" tekst="Laster avtaler..." />;
   }
 
-  if (avtaler.length === 0) {
-    return <Alert variant="info">Fant ingen avtaler</Alert>;
-  }
-
   return (
     <div className={classNames(styles.tabell_wrapper, styles.avtaletabell)}>
       <div className={styles.tabell_topp_container}>
-        <PagineringsOversikt
-          page={page}
-          antall={avtaler.length}
-          maksAntall={pagination?.totalCount}
-          type="avtaler"
-          antallVises={filter.antallAvtalerVises}
-          setAntallVises={(value) => {
-            resetPaginering(setPage);
-            setFilter({ ...filter, antallAvtalerVises: value });
-          }}
-        />
+        <div>
```

Comment:
Trengs denne diven?

Reply:
Ja, for å kunne ha mine og dropdown for antall på venstresiden og last ned excel-knapp på høyre siden. Det blir 2 flex children istedenfor tre (som ville flyttet vis mine checkbox til midten av skjermen)